data.sql

```diff
@@ -35,4 +35,76 @@ INSERT INTO
     neutered,
     weight_kg
   )
-VALUES ('Devimon', 'May 12, 2017', 5, TRUE, 11);
\ No newline at end of file
+VALUES ('Devimon', 'May 12, 2017', 5, TRUE, 11);
+
+
+INSERT INTO
+  animals (
+    NAME,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Charmander', 'February 8, 2020', 0, TRUE, -11);
+
+INSERT INTO
+  animals (
+    NAME,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Plantmon', 'November 15, 2021', 2, TRUE, -5.7);
+
+INSERT INTO
+  animals (
+    NAME,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Squirtle', 'April 2, 1993', 3, FALSE, -12.13);
+
+INSERT INTO
+  animals (
+    NAME,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Angemon', 'June 12, 2005', 1, TRUE, 45);
```

Comment:
- [ ] I guess you made a mistake here. The weight for the animal `Angemon` should be a negative value `-45`

Reply:
not resolved

Reply:
Okay, I did fix it right now

Reply:
you can submit now for approval if you've corrected it